base/shell/explorer/trayprop.cpp

```diff
@@ -270,6 +227,108 @@ class CStartMenuSettingsPage : public CPropertyPageImpl<CStartMenuSettingsPage>
     }
 };
 
+class CNotifySettingsPage : public CPropertyPageImpl<CNotifySettingsPage>
+{
+private:
+    HBITMAP m_hbmpTray;
+    HWND m_hwndTaskbar;
+
+    void UpdateDialog()
```

Comment:
Maybe use some sort of distinctive name for private functions, like `_UpdateDialog`, as it is already done elsewhere in the shell code.

Reply:
Done. Since very similar classes exist for the other property pages, I updated the names of their `UpdateDialog` methods also.